models/order.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+module.exports = (sequelize, DataTypes) => {
+  const order = sequelize.define(
+    "order",
+    {
+      order_id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      customer_id: DataTypes.INTEGER,
+      order_data: DataTypes.DATE,
+      total_price: DataTypes.FLOAT
+    },
+    {tableName: "order",
+    timestamps: false,}
+  );
+  order.associate = function(models) {
+    // associations can be defined here
+  };
+  sequelize.sync();
```

Comment:
Why sequelize.sync()?

Reply:
i was supposed to run db:migrate but i was not sure whether it would remove the current data. So to be one the safer side i used sequelize.sync()